Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine
+
+
+WORKDIR /usr/src/hackerrank-testcase-generator
+
+COPY ./ ./
+
+RUN npm install
+
+
```

Comment:
Also, add env vars too.
`ENV MongoURI here`

Reply:
Updated the Docker file